### DISCUSSION TITLE
2023/02/05/My-first-blog/

### DISCUSSION THREAD

**Discussion by yuhiri-s-giscus** [talentestors/talentestors.github.io#55] [Announcements]
# 2023/02/05/My-first-blog/

https://yuhiri.me/2023/02/05/My-first-blog/

**Comment by talentestors** [comment]
# giscus Test

![nahida](https://s2.loli.net/2023/10/23/BfgS7akWcYNFhqT.jpg)

## vim

输入so，重新载入vim配置

```cpp
string str = "talentestors"
```

**Reply by talentestors** [reply]
Test
